common/lib/xmodule/xmodule/word_cloud_module.py

```diff
@@ -247,6 +253,7 @@ def get_html(self):
             'element_id': self.location.html_id(),
             'num_inputs': self.num_inputs,
             'submitted': self.submitted,
+            'js_includes': js_includes,
```

Comment:
Could we convert this `get_html()` into a `student_view()` and return a `Fragment`, so that the runtime can be smarter and we don't get repeated JS if there are multiple problems of this type in a sequence?